Userland/Libraries/LibJS/Interpreter.cpp

```diff
@@ -180,5 +196,4 @@ LexicalEnvironment* Interpreter::current_environment()
     VERIFY(is<LexicalEnvironment>(vm().call_frame().scope));
     return static_cast<LexicalEnvironment*>(vm().call_frame().scope);
 }
-
```

Comment:
Nitpick: removing that seems pointless, let's keep an empty line before the closing namespace curly :)